### PR TITLE
#158429783 Create landing page with signup form

### PR DESCRIPTION
**Pull request by JoshuaOndieki**
**What does this PR do?**
Adds a landing page UI for the Ride My Way app project.

**Description of Task to be completed?**
The landing page which visitors interact with before signing in/up. The page contains a form for visitors to quickly and easily sign up to the site.

**Tasks achieved in this PR**

- [x] Landing page

- [x] Signup interface in the landing page.

**How should this be manually tested?**
Clone this repo and test locally on your computer.

**Background Context?**
The UI has no functionality.

**Resources & Libs**
[This Pivotal Tracker board](https://www.pivotaltracker.com/n/projects/2179581) was used to plan and manage the tasks.

**What are the relevant Pivotal Tracker stories?**
#158429783

**Screenshots**
screenshot from 2018-06-19 19-23-11

### CODE REVIEW

File: UI/static/css/landingpage.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    height: 60%;
+    width: 35%;
+    background-color: cyan;
+    display: inline-block;
+    margin-left: 20%;
+    vertical-align: middle;
+    align-content: center;
+}
+
+.call-to-action{
```
Comment (xcixor) [inline UI/static/css/landingpage.css]: Great job using classes to style elements. Makes it easy to know the group of elements that belong together

**Comment by xcixor** [comment]
Great progress but maybe you can work on the colors a bit to enhance the look and feel..my thoughts though.

**Comment by JoshuaOndieki** [comment]
@xcixor do you have any color in mind that I can swap?

File: UI/landingpage.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            <input type="text" placeholder="username@example.com" name="email" required>
+
+            <input type="text" name="username" placeholder="username" required>
+
+            <select name="usertype" required>
+              <option value="driver">Driver</option>
+              <option value="passenger">Passenger</option>
+            </select>
+
+            <input type="password" placeholder="******" name="password" required>
```
Comment (boswellgathu) [inline UI/landingpage.html]: please don't add the asterisk as a placeholder, the can confuse a person, just say password or any other description

File: UI/static/css/main.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+.logo{
+    background: cyan;
+    border-radius: 50%;
+    height: 100px;
+    line-height: 100px;
+    text-align: center;
+    vertical-align: middle;
+    width: 100px;
+    display: inline-block;
+
```
Comment (boswellgathu) [inline UI/static/css/main.css]: Make sure to format your css properly, don't leave blank lines everywhere this makes the file look untidy and tiring to read. Also mind spacing, there should be a space between a class name and the opening curly brace.

**Review by boswellgathu — COMMENTED** [review]
You have made good progress. I just have a few things you should look at

**Comment by xcixor** [comment]
I must admit I'm also not that good with colors but as a suggestion, think of the mood or feel your application intends to convey then visit [this site](https://color.adobe.com/explore/?filter=most-popular&time=month), type in an adjective for example 'delightful' and it gives you a list of color themes that you can use on your site

**Comment by evamaina** [comment]
Nice one

**Pull request merged**